Source/ROLib/Modules/ROLAnimateEngineHeat.cs

```diff
@@ -124,13 +129,27 @@ private void locateAnimatedTransforms()
             if (animatedRenderers == null || animatedRenderers.Length == 0) { print("ERROR: Could not locate any emissive meshes for name: " + meshName); }
         }
 
+        private void locateSolverEngines()
+        {
+            ModuleEngines solverEnginesModule = ROLModInterop.getSolverEngineModule(part, engineID);
+
+            if (solverEnginesModule == null)
+                return;
+
+            MethodInfo getter = ROLModInterop.getSolverEngineTempProperty().GetGetMethod();
+
+            getSolverChamberTemp = (Func<double>) Delegate.CreateDelegate(typeof(Func<double>), solverEnginesModule, getter);
+
+            useSolverEngines = true;
```

Comment:
This is fine.  But along with the above, I tend to be a fan of using Properties here, and also making use of the fact that you have the getSolverChamberTemp delegate, or it is null.

public bool useSolverEngines => getSolverChamberTemp != null;
(or getSolverChamberTemp is Func<double>, I guess!)

Reply:
Pretty sure null checking isn't necessery, the checks done beforehand are enough to ensure that `getSolverChamberTemp` isn't null. If there is an error in the creation process (although I don't think that is possibly) it will result in an exception and jump out of the method

Reply:
And in that case `useSolverEngines` will be false anyways

Reply:
Actually, looking through the documentation, I don't think `Delegate.CreateDelegate` ever returns null, if there is something wrong it will throw an exception